feature-detects/hbbtv/video.js

```diff
@@ -0,0 +1,524 @@
+/* global setTimeout */
+/*!
+{
+  "name": "HbbTV Video",
+  "property": "hbbtvvideo",
+  "tags": ["hbbtv", "oipf", "video"],
+  "async" : true,
+  "warnings": [
+    "This test is currently only compatible to HbbTV >= 1.x",
+    "All siblings of an existing HbbTV video object in the DOM will be deleted during the test"
```

Comment:
The problem is that HbbTV < 2.x only allows one video object to be inside the DOM and so I have to remove the old video object.
Do you think it would be ok to tell the user that the test requires that there is no other video object in the DOM during the test? It would reduce the amount of code a lot.